chemistry/rosetta/pose.py

```diff
@@ -58,18 +58,17 @@ def load(pose):
                 except KeyError:
                     raise RosettaError('Could not recognize element: %s.'
                                        % atsym)
+
+                params = dict(atomic_number=atomic_number, name=atname,
+                              charge=0.0, mass=mass, occupancy=0.0,
+                              bfactor=0.0, altloc='', number=atnum,
+                              rmin=rmin, epsilon=epsilon)
+
                 if atinfo.is_virtual():
-                    atom = ExtraPoint(atomic_number=atomic_number,
-                                      name=atname, charge=0.0, mass=mass,
-                                      occupancy=0.0, bfactor=0.0,
-                                      altloc='', number=atnum, rmin=rmin,
-                                      epsilon=epsilon)
+                    atom = ExtraPoint(**params)
                 else:
-                    atom = Atom(atomic_number=atomic_number, name=atname,
-                                charge=0.0, mass=mass, occupancy=0.0,
-                                bfactor=0.0, altloc='', number=atnum,
-                                rmin=rmin, epsilon=epsilon)
-                atom.xx, atom.xy, atom.xz = tuple(at.xyz())
```

Comment:
Did you mean to get rid of this line?  I don't see where you set the atomic coordinates anywhere else (apologies if I'm just blind)